src/query/storages/fuse/src/operations/commit.rs

```diff
@@ -123,37 +124,42 @@ impl FuseTable {
                         Ok(())
                     };
                 }
-                Err(e) if self::utils::is_error_recoverable(&e, transient) => match backoff
-                    .next_backoff()
-                {
-                    Some(d) => {
-                        let name = tbl.table_info.name.clone();
-                        debug!(
-                            "got error TableVersionMismatched, tx will be retried {} ms later. table name {}, identity {}",
-                            d.as_millis(),
-                            name.as_str(),
-                            tbl.table_info.ident
-                        );
-                        common_base::base::tokio::time::sleep(d).await;
-                        latest = tbl.latest(ctx.as_ref(), catalog_name).await?;
-                        tbl = FuseTable::try_from_table(latest.as_ref())?;
-                        retry_times += 1;
-                        continue;
-                    }
-                    None => {
-                        info!("aborting operations");
-                        let _ = self::utils::abort_operations(ctx.as_ref(), operation_log).await;
-                        break Err(ErrorCode::OCCRetryFailure(format!(
-                            "can not fulfill the tx after retries({} times, {} ms), aborted. table name {}, identity {}",
-                            retry_times,
-                            Instant::now()
-                                .duration_since(backoff.start_time)
-                                .as_millis(),
-                            tbl.table_info.name.as_str(),
-                            tbl.table_info.ident,
-                        )));
+                Err(e) if self::utils::is_error_recoverable(&e, transient) => {
```

Comment:
OpenDAL has its logic for backoff. And I have a plan to refactor here. So we can return directly here to make things simpler.